tests/compass-runtime-agent/test/testkit/assertions/k8s_resource_checker.go

```diff
@@ -219,6 +225,12 @@ func (c *K8sResourceChecker) assertAssetGroup(t *testing.T, apiPackage *graphql.
 			source, found := getAPISource(eventAPI.ID, assetGroup)
 			require.True(t, found)
 
+			eventApiDescription := ""
+			if eventAPI.Description != nil {
+				eventApiDescription = *eventAPI.Description
+			}
+			assert.Equal(t, eventApiDescription, source.DisplayName)
```

Comment:
I'm not sure if above comments yield additional changes here...

Reply:
Done